### DISCUSSION TITLE
Just can find this? Customize the Group Buttons

### DISCUSSION THREAD

**Discussion by metacraft-hq** [yajra/laravel-datatables#2783] [General]
Hi, I have spent the better part of 2 days customizing the table look and feel -- the last area I just cant seem to find where or how to is the `Group Buttons` from `Select All` to `Delete Selection` 

See IMAGE for visual :

![image](https://user-images.githubusercontent.com/89227642/168102325-92301691-14f2-447c-9267-945c527f3d4d.png)

I want to add padding far left and in-between, and group position them center  and finally  color to each?

**Comment by mx-advaud** [comment]
Check this 
https://github.com/yajra/laravel-datatables/issues/483

**Reply by metacraft-hq** [reply]
Thanks - but i had to import that file to edit the views .. I am trying to find where i can add the padding, button colors etc?
